scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -474,7 +474,8 @@
                                         "enabled": true,
                                         "config": {
                                             "logAnalyticsWorkspaceResourceID": "[parameters('workspaceResourceId')]",
-                                            "useAADAuth": "true"
+                                            "useAADAuth": "true",
+                                            "enableRetinaNetworkFlags": "false"
```

Comment:
value of enableRetinaNetworkFlags should be paratmer driven not hard coded to "false".

Reply:
added